messenger.rst

```diff
@@ -73,26 +100,58 @@ Once you've created your handler, you need to register it:
 Transports
 ----------
 
-The communication with queuing system or third parties is delegated to
+The communication with queuing systems or third parties is delegated to
 libraries for now. The built-in AMQP transport allows you to communicate with
 most of the AMQP brokers such as RabbitMQ.
 
 .. note::
 
-    If you need more message brokers, you should have a look to `Enqueue's transport`_
+    If you need more message brokers, you should have a look at `Enqueue's transport`_
     which supports things like Kafka, Amazon SQS or Google Pub/Sub.
 
 A transport is registered using a "DSN", which is a string that represents the
 connection credentials and configuration. By default, when you've installed
 the messenger component, the following configuration should have been created:
 
-.. code-block:: yaml
-
-    # config/packages/messenger.yaml
-    framework:
-        messenger:
-            transports:
-                amqp: "%env(MESSENGER_DSN)%"
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/messenger.yaml
+        framework:
+            messenger:
+                transports:
+                    amqp: "%env(MESSENGER_DSN)%"
```

Comment:
@sroze Can you have a look here? From looking at the code I would assume that this would never work as a string would be transformed into an empty array.